defopt.py

```diff
@@ -12,6 +12,7 @@
 from argparse import (SUPPRESS, ArgumentParser, RawTextHelpFormatter,
                       ArgumentDefaultsHelpFormatter)
 from collections import defaultdict, namedtuple, OrderedDict
+import contextlib
```

Comment:
These are split into separate subgroups for `import x` and `from x import y`. I was aiming for what you get from isort, though I'm not actually running the tool so this may differ slightly.